igmarkets.go

```diff
@@ -537,7 +537,7 @@ func (ig *IGMarkets) GetPriceHistory(epic, resolution string, max int, from, to
 		limitStr = fmt.Sprintf("&max=%d", max)
```

Comment:
max is set only when "to" or "from" has no time.Time value.

GetPriceHistory requires time.Time so it will never set "max" in the "else if" to 1.
Setting max is required to not get the default of i think 10.

Reply:
What I mean is: If `max > 0`, we set `max` twice. In `page` and in `limitStr`. This is wrong IMO because both are used to assemble the URL. I think we need to removed it from `page` and leave `limitStr` empty if `max` is zero.